sdk/messaging/src/main/java/com/klaviyo/messaging/Utils.kt

```diff
@@ -50,6 +50,11 @@ internal fun decodeWebviewMessage(webMessage: String): KlaviyoWebFormMessageType
                 )
             }
         }
+        IAF_MESSAGE_TYPE_ABORT -> {
```

Comment:
Wanted to be able to test aborting while I'm implementing the fender side, so i threw this in, doesn't have to be the permanent implementation